utils/bundles.stanza

```diff
@@ -29,39 +29,35 @@ public pcb-bundle power :
 public pcb-bundle fpga-diff : 
   diff-pair-pins()
 
-public pcb-bundle jtag :
+public pcb-enum ocdb/bundles/JTAGPins :
+  TRSTN
```

Comment:
Since enum variants are global, they should be prefixed. The convention we use is `NameOfEnumNameOfVariant`, in this case:

```public pcb-enum ocdb/bundles/JTAGPins:
  JTAGTRSTN
```

That said we should probably notuse single variant enums, a boolean as a generator argument would work fine. Is this enum going to be expanded later?